### DISCUSSION TITLE
Add Matrix space

### DISCUSSION THREAD

**Discussion by MagicLike** [JulianPrieber/littlelink-custom#153] [Feature requests]
For better privacy it would be great to have a [Matrix](https://matrix.org) space / room.
Eventually it could be mirrored with Discord.

@JulianPrieber If it is ok, I would create it and add you then.

**Comment by JulianPrieber** [comment]
Hey @MagicLike thanks for the offer!

I read through their website and really like the concept. I think this would be a good addition/alternative to our Discord server.

I really would like to host a Matrix homeserver. So far, I spinned up a Debian container and ran the setup. I couldn't get it to work, and I don't really know when I can find the time to do so.

I might get something up and running over the weekend.

**Reply by MagicLike** [reply]
Nice!
(If it doesn't work, we could just set up the space on the official matrix.org server.)

**Reply by JulianPrieber** [reply]
> (If it doesn't work, we could just set up the space on the official matrix.org server.)

I'll probably do that first just to figure some stuff out.

**Reply by MagicLike** [reply]
But when it works you can setup the [Discord bridge](https://github.com/matrix-org/matrix-appservice-discord) right away with it!

**Reply by MagicLike** [reply]
Also if you need help, my Matrix handle is `@magiclike:matrix.org`.

**Reply by MagicLike** [reply]
Any updates on the space?

**Reply by JulianPrieber** [reply]
So, I tried setting up the Discord bridge, but it won't really work.

**Reply by MagicLike** [reply]
Oh no, have you opened an issue at the repo? Maybe they know what is wrong...

**Reply by JulianPrieber** [reply]
I'm trying to get this one working, https://t2bot.io/discord/. 
This seemed to be a fairly easy solution for this, but the discord bot won't link.

According to their website, the discord bot is up.

I might have to host a bridge bot myself, if I do that I will try to host everything myself.

**Reply by JulianPrieber** [reply]
Its working now. I had end-to-end encryption on in the room, even tought I had turned it off.

**Reply by MagicLike** [reply]
Nice! Is it possible for you invite me to the Matrix Space, to test some stuff?

**Comment by JulianPrieber** [comment]
You seem to be fairly involved in this project, I would love to have you on our beta tester team!

Would you care to join us?

**Reply by MagicLike** [reply]
Shure, I'd really like that! 😃

**Reply by JulianPrieber** [reply]
Alright! Here's a page that pretty much describes the basics: https://littlelink-custom.com/beta-tester

We are a small team that works on developing new versions and brainstorm ideas.
And well... beta testing.

**Reply by JulianPrieber** [reply]
The next update will add among other things custom code and JS support to themes.
https://demo.littlelink-custom.com/@polygon

**Reply by MagicLike** [reply]
I will make a fresh install in the next week or so and would love to start beta testing!

**Reply by JulianPrieber** [reply]
### Sounds good 👍

If everything goes well, I'll release the next update this weekend. The thing is pretty much finished, and I'm currently working on writing documentation for it.

The thing should be safe to download to your main instance if you want to use it early.

**How to make your own theme v2:**

https://github.com/JulianPrieber/llc-themes/tree/v2/contributing
https://github.com/JulianPrieber/llc-default/tree/v2

_Example theme:_
[polygon.zip](https://github.com/JulianPrieber/littlelink-custom/files/9538297/polygon.zip)

**Reply by JulianPrieber** [reply]
Oh, yeah and this of course: https://demo.littlelink-custom.com/@yes

[yes.zip](https://github.com/JulianPrieber/littlelink-custom/files/9538312/yes.zip)
